parsers/DK.py

```diff
@@ -4,8 +4,8 @@
 
 from requests import Response, Session
 
-from .lib.config import refetch_frequency
-from .lib.exceptions import ParserException
+from electricitymap.contrib.parsers.lib.config import refetch_frequency
+from electricitymap.contrib.parsers.lib.exceptions import ParserException
```

Comment:
Pretty sure this will fail on windows as it don't support symbolic links.

Reply:
In which folder should I be ? Because otherwise the relative path doesn't work if I'm at the root, `poetry run parsers/NL.py` it fails due to the relative imports.

Reply:
I think it's best to use the `test_parser` instead of running the file directly.

So try `poetry run test_parser NL`, that should work.